pytest_splunk_addon/plugin.py

```diff
@@ -116,6 +120,62 @@ def pytest_sessionstart(session):
         sample_generator.get_samples(store_events)
 
 
+def pytest_sessionfinish(session, exitstatus):
+    try:
+        with open(f".{os.sep}splunk_type.txt") as splunk_type_file:
+            splunk_data = [line.rstrip() for line in splunk_type_file]
+            splunk_type = splunk_data[0]
+            splunk_keep_alive = splunk_data[1]
+            splunk_app_path = splunk_data[2]
+        if (
+            (os.environ.get("PYTEST_XDIST_WORKER") == None)
+            and (splunk_type == "kubernetes")
+            and (splunk_keep_alive != "True")
+        ):
+            LOGGER.info("SessionFinish - destroying all kubernetes resources")
+            parser = conf_parser.TABConfigParser()
+            parser.read(os.path.join(f"{splunk_app_path}", "default", "app.conf"))
+            splunk_addon_name = parser.get("package", "id")
+            os.environ["NAMESPACE_NAME"] = splunk_addon_name.replace("_", "-").lower()
+            files = [
+                f".{os.sep}exposed_splunk_ports.log",
```

Comment:
Do we really need to have `.{os.sep}` before file name? You are then iterating over those files and deleting those if they exist, just file name should work.

Reply:
Updated code by removing extra .{os.sep}  before file names.